ssm/ssm2.py

```diff
@@ -127,12 +127,27 @@ def set_dns(self, dn_list):
     ##########################################################################
     # Methods called by stomppy
     ##########################################################################
-            
-    def on_send(self, headers, unused_body):
+
+    def on_send(self, frame, unused_body=None):
         '''
         Called by stomppy when a message is sent.
+
+        unused_body is only present for to have a backward compatible
+        method signiture when using stomp.py v3.1.X
         '''
-        log.debug('Sent message: %s', headers['empa-id'])
+        try:
+            # Try the stomp.py v4 way first
+            empaid = frame.headers['empa-id']
+        except KeyError:
+            # Then you are most likely using stomp.py v4.
+            # on_send is now triggered on non message frames
```

Comment:
What does this mean for us? We'll be getting "messages" being logged for CONNECT frames and such?

Reply:
yes, but only logged at the `debug` level

Reply:
`stomp.py` also currently logs such messages at the `debug` level. So they would be some duplication of `debug` logs

Reply:
What do the non-message frames look like?

I guess our debug is logging the empa-id header specifically. For comparison, what does stomp.py log?

Reply:
The frame looks like `{cmd=STOMP,headers=[{'accept-version': '1.1'}],body=}`

stomp.py logs it as 
```
2018-05-16 14:11:15,179 - stomp.py - INFO - Sending frame: 'STOMP', headers={'accept-version': '1.1'}
```